backend/app.js

```diff
@@ -4,6 +4,8 @@ var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 var cors = require("cors");
+var mongoose = require('mongoose');
+var Item = require('./models/item');
```

Comment:
Can these be changed from const to var since they shouldn't be changed later in the code? Can you also do it for the variables above (I know you didn't write most of these variables).